src/smart-components/ChannelList/components/ChannelPreview/utils.js

```diff
@@ -25,16 +25,17 @@ export const getChannelTitle = (channel = {}, currentUserId, stringSet = LabelSt
 
 export const getLastMessageCreatedAt = (channel, locale) => {
   const createdAt = channel?.lastMessage?.createdAt;
+  const thirdParam = locale ? { locale } : null;
```

Comment:
Nit, dont need to rewrite -> Calling it options is better
<img width="522" alt="Screen Shot 2022-03-28 at 1 59 58 PM" src="https://user-images.githubusercontent.com/11382805/160329468-b7c30d89-3a2f-4648-9fe2-5b51839e5cc2.png">